src/scripts.js

```diff
@@ -81,57 +70,80 @@ let recipe = new Recipe(recipeData);
 //if the target id === event.target.parentNode.id
 
 
-    
+allRecipesView.addEventListener('click', event => {
```

Comment:
We're gonna need to refactor our event listener to try to make it a little more condensed.
But the functionality is awesome!